src/classes/VOL_SharedCode.cls

```diff
@@ -819,7 +819,7 @@ global with sharing class VOL_SharedCode {
                 Schema.SObjectField fld = mapFieldDesc.get(strField.replace(StrTokenNSPrefix(''), ''));     
                 if (fld != null && !fld.getDescribe().isAccessible()) {
                     ApexPages.addMessage(new ApexPages.Message(ApexPages.Severity.FATAL, 'Field ' + strObj + '.' + strField +
-                        ' needs to have field level security enabled on the Site\'s Guest User profile. Please see the V4S Setup & Configuration Guide.'));
+                        ' You need to enable field level security for this field on the Site&apos;s Guest User profile. For more information, see the &lt;a target=&apos;_blank&apos; href=&apos;https://powerofus.force.com/V4S_Config_Guide&apos;&gt;Set Up and Configure Volunteers for Salesforce Guide&lt;/a&gt;.'));

```

Comment:
@sam-knox - Something wonky with the way this error is being displayed... 
![screen shot 2018-12-04 at 10 19 16 am](https://user-images.githubusercontent.com/450473/49456200-40e63900-f7ae-11e8-9fb3-e106d64e594d.png)

Reply:
@davisagli @jjbennett - Gotcha. I was wondering about that. Let me make a change and resubmit.

Reply:
Looks better, this is the screen when we have multiple missing fields.
![screen shot 2018-12-04 at 12 42 18 pm](https://user-images.githubusercontent.com/450473/49465050-33d34500-f7c2-11e8-9292-0d4fab144753.png)